internal-docs/engineering/on-call-rotation.md

```diff
@@ -0,0 +1,80 @@
+- A feature team may release buggy code using Paste, and we may get pinged as part of an incident
+- We may mislabel our SEMVER versioning and result in a breaking change by an implementation team
+- We may experience a malicious package takeover on NPM / of our NPM packages
+- We may experience a malicious contributor to our GH repo which may have negative repercussions, including Actions mishaps
+- Paste data may stop working, resulting in no data being sent to our reporting service
+
+[More detailed failure modes and disaster recovery strategies here.](https://docs.google.com/document/d/18iuTYuiwnA9YZ-7AN9lU1i-3mzcY1VcgzeAtIoiq40c/edit)
+
+## Where can I find the schedule for on-call rotation?
+
+Since the On-Call rotation is an Engineering-only rotation, schedules for on-call will likely not align with our team’s helper duty rotation. On-call rotation has its own schedule, and you can find it on the [PagerDuty website](https://twilio.pagerduty.com/) or on the PagerDuty mobile app.
+
+PagerDuty will send out emails, and if configured, push notifications through the mobile app, ahead of your rotation assignments to let you know when you have an assigned role.
+
+## What are the roles for on-call rotation?
+
+- **On-Call:** The person who is the first contact in the case of an incident. Respond within 5 minutes is the goal.
+- **On-Call backup:** The person who will be contacted during an incident if the on-call person does not acknowledge the incident within a 10 minute time window.
+- **Escalation responder:** Engineering manager, who will be contacted during an incident if both the on-call Engineers fail to acknowledge the incident after an additional 10 minutes.
+- **Secondary escalation responder:** Product owner, who will be contacted during an incident if the Escalation responder fails to acknowledge the incident within the next 30 minutes.
+
+| On-Call notified | Backup notified | Escalation responder notified | Secondary escalation responder notified |
+| ---------------- | --------------- | ----------------------------- | --------------------------------------- |
+| 0m               | 10m (+10m)      | 20m (+10m)                    | 50m (+30m)                              |
+
+## How do I best prepare for my rotation?
+
+- Make sure you’ve completed the PagerDuty training on Continu.
+- Check your [PagerDuty](https://twilio.pagerduty.com/) profile and ensure your contact information is up to date.
+- Get the [PagerDuty app](https://apps.apple.com/us/app/pagerduty/id594039512) set up on your phone, and follow processes to ensure that PagerDuty notifications will be classified as time-sensitive to bypass your phone’s do not disturb settings.
+- You can use the PagerDuty Slack integration to check who is on call at any point by typing ‘/fh help’ and hitting enter.
+- Review our [TODO: incident runbooks] to have a solid understanding of common production incidents and how to triage them.
```

Comment:
Is this still a TODO or do we have it now? Is it the doc shadi made?

Reply:
Runbooks are different. We only have 1 or 2 so far https://drive.google.com/drive/u/0/folders/1-_Xz9xtQ-NUA9kjNqw328g5OKH02swpa
There are tickets for more